src/Master.cairo

```diff
@@ -464,42 +417,25 @@ mod Master {
             self._contributions.write(old_address, zero_contribution);
             self._contributions.write(new_address, contribution);
```

Comment:
wouldn't it be safer to update the state after finishing the migration (after line425)?